site/docs/latest/getting-started/Pulsar-2.0.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Pulsar 2.0
+hide_pulsar2_notification: true
+new: true
+tags: ["2.0", "tenants", "clients"]
+---
+
+Pulsar 2.0 is a major new release for Pulsar that brings some bold changes to the platform, including [simplified topic names](#topic-names)
+
+## New features in Pulsar 2.0
+
+Feature | Description
+:-------|:-----------
+[Pulsar Functions](../../functions/overview) | A lightweight compute option for Pulsar
+<!-- [Topic compaction](../../cookbooks/compaction) | TODO -->
+
+## Major changes
+
+There are a few major changes that you should be aware of, as they may significantly impact your day-to-day usage.
+
+### Properties versus tenants {#tenants}
+
+Previously, Pulsar had a concept of {% popover properties %}. A property is essentially the exact same thing as a {% popover tenant %}, so the "property" terminology has been removed in version 2.0. The [`pulsar-admin properties`](../../CliTools#pulsar-admin) command-line interface, for example, has been replaced with the [`pulsar-admin tenants`](../../CliTools#pulsar-admin-tenants) interface. In some cases the properties terminology is still used but is now considered deprecated and will be removed entirely in a future release.
+
+### Topic names
+
+Prior to version 2.0, *all* Pulsar topics had the following form:
+
+{% include topic.html type="{persistent|non-persistent}" ten="property" n="namespace" c="cluster" t="topic" %}
+
+Two important changes have been made in Pulsar 2.0:
+
+* There is no longer a [cluster component](#no-cluster)
+* Properties have been [renamed to tenants](#tenants)
+* You can use a [flexible](#flexible-topic-naming) naming system to shorten many topic names
+
+#### No cluster component {#no-cluster}
+
+The {% popover cluster %} component has been removed from topic names. Thus, all topic names now have the following form:
+
+{% include topic.html type="{persistent|non-persistent}" ten="tenant" n="namespace" t="topic" %}
```

Comment:
"..Existing topics that are using the legacy name format will continue to work without any change. There is no plan to remove support for that."